src/index.js

```diff
@@ -10,3 +11,4 @@ module.exports.crypto = crypto
 module.exports.amino = amino
 module.exports.utils = utils
 module.exports.ledger = Ledger
+module.exports.rpcClient = rpcClient
```

Comment:
What do you think of `rpc` as an export name. Kind of follows the same pattern as the rest, yes?

Reply:
agree